sentry/src/main/java/io/sentry/internal/gestures/UiElement.java

```diff
@@ -0,0 +1,59 @@
+package io.sentry.internal.gestures;
+
+import java.lang.ref.WeakReference;
+import java.util.Objects;
```

Comment:
`h`: I think we cannot use `java.util.Objects` because it's available on Android API 19+, and our minSdkVersion is still 14. So we should use `io.sentry.util.Objects` and add missing methods there.

Also while typing I just realized that we have used it in other places already, so we should fix that there as well, but not in this PR :)